beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -401,8 +521,9 @@ func scoreByWeight(weight, threshold float64) float64 {
 
 // maxScore attainable by a peer.
 func maxScore() float64 {
-	totalWeight := beaconBlockWeight + aggregateWeight + attestationTotalWeight +
-		attesterSlashingWeight + proposerSlashingWeight + voluntaryExitWeight
+	totalWeight := beaconBlockWeight + aggregateWeight + +syncContributionWeight +
```

Comment:
what does the `+ +` do? Does it make it a positive number?

Reply:
That is actually a bug, I a more surprised that was valid go syntax and the compiler was fine with it... . In any case I have fixed it, nice spot.